### PR TITLE
Fixed redundant else statement

### LINKED ISSUE #11720: daemon/graphdriver/devmapper/deviceset.go redundant else
In `GetDeviceStatus`:
```
	if sizeInSectors, mappedSectors, highestMappedSector, err := devices.deviceStatus(info.DevName()); err != nil {
		return nil, err
	} else {
		status.SizeInSectors = sizeInSectors
		status.MappedSectors = mappedSectors
		status.HighestMappedSector = highestMappedSector
	}
```

### PR DESCRIPTION
**Pull request by aneshas**
Fixed redundant else in GetDeviceStatus issue #11720 
Closes #11720 

Signed-off-by: Anes Hasicic anes.hasicic@gmail.com

### CODE REVIEW

**Comment by duglin** [comment]
@aneshas can you add "Closes #11720" to your initial comment so that github will automatically close the original issue?

**Comment by duglin** [comment]
@aneshas you can see the compilation errors in the 'janky' log above:
---> Making bundle: binary (in bundles/1.5.0-dev/binary)
\# github.com/docker/docker/daemon/graphdriver/devmapper
daemon/graphdriver/devmapper/deviceset.go:1556: undefined: sizeInSectors
daemon/graphdriver/devmapper/deviceset.go:1557: undefined: mappedSectors
daemon/graphdriver/devmapper/deviceset.go:1558: undefined: highestMappedSector
Build step 'Execute shell' marked build as failure

File: daemon/graphdriver/devmapper/deviceset.go [hunk 1]
```diff
@@ -1551,12 +1551,12 @@ func (devices *DeviceSet) GetDeviceStatus(hash string) (*DevStatus, error) {
 
 	if sizeInSectors, mappedSectors, highestMappedSector, err := devices.deviceStatus(info.DevName()); err != nil {
```
Comment (LK4D4) [inline daemon/graphdriver/devmapper/deviceset.go]: This not compiling :) You need to split assignment and `if err != nil` to separate lines.

**Comment by LK4D4** [comment]
@aneshas oooh, something pretty bad with your commits.

**Comment by LK4D4** [comment]
@aneshas You can do this without creating new PR. Just need to learn a little about `git rebase -i` and all stuff about changing local tree. I'm okay if you don't want to learn it just for this PR, but it is pretty interesting and useful, so I suggest you to read about it when you have time.

**Comment by aneshas** [comment]
@LK4D4 Yes, thank you I will, but I will create a new pr for this one anyway after I read that stuff. Thanks

**Comment by LK4D4** [comment]
@aneshas note, that it will be impossible to reopen it after force push :)

**Pull request closed without merging**